skyVegasTests/stakeUpdateGameClient.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test, type Page } from '@playwright/test';
+import {
+    launchGame,
+    login,
+    validateConsoleMessages,
+    startEventListener,
+    readGames,
+} from './utilities';
+import { ExpectedMessage } from '../types/expectedMessage';
+
+let page: Page;
+readGames().forEach((game) => {
+    const consoleMessages: string[] = [];
```

Comment:
Let's move initialising the consoleMessages inside the test, between lines 25 and 26. This will ensure that if multiple tests are added to this file in the future , each tests will start with a fresh , empty console message array.